src/cbc_sdk/endpoint_standard/base.py

```diff
@@ -62,7 +61,7 @@ def _update_entire_object(self):
             new_object_info = deepcopy(self._info)
             try:
                 if not self._new_object_needs_primary_key:
-                    del(new_object_info[self.__class__.primary_key])
+                    del (new_object_info[self.__class__.primary_key])
```

Comment:
Are the parentheses needed/helping?

Reply:
They certainly aren't harming anything.  The extra space was probably added by the auto formatter.